meson.build

```diff
@@ -75,9 +75,26 @@ if get_option('buildtype') == 'debug'
   add_project_arguments('-DHYPRLAND_DEBUG', language: 'cpp')
 endif
 
+globber = run_command('find', 'src', '-name', '*.h*', check: true)
+headers = globber.stdout().strip().split('\n')
+foreach file : headers
+  install_headers(file, subdir: 'hyprland', preserve_path: true)
+endforeach
+
 subdir('protocols')
 subdir('src')
 subdir('hyprctl')
 subdir('assets')
 subdir('example')
 subdir('docs')
+
+pkgconfig_configuration = configuration_data()
+pkgconfig_configuration.set('PREFIX', get_option('prefix'))
+pkgconfig_configuration.set('HYPRLAND_VERSION', meson.project_version())
+pkg_install_dir = join_paths(get_option('datadir'), 'pkgconfig')
+configure_file(
+	input: 'hyprland.pc.in',
+	output: 'hyprland.pc',
+	configuration: pkgconfig_configuration,
+	install_dir: pkg_install_dir,
+)
```

Comment:
I actually did this at first but decided to use `hyprland.pc.in` so that CMake could also build `hyprland.pc`. But I suppose I could let CMake do its thing, while having Meson generate the pc file directly.